ote_sdk/ote_sdk/test_suite/training_tests_actions.py

```diff
@@ -390,8 +399,10 @@ def _run_ote_pot(
         assert (
             self.optimized_model_pot.model_format == ModelFormat.OPENVINO
         ), "Wrong model format after pot"
+        #TO DO: Model.OptimizationType is "int enum" class, and OptimizationType is "enum",
```

Comment:
let's way for the fix, don't fix it this way

Reply:
Changed, already fixed in https://github.com/openvinotoolkit/deep-object-reid/pull/132